tests/gnrc_ipv6_ext/README.md

```diff
@@ -0,0 +1,43 @@
+# `gnrc_ipv6_ext` test
+
+This test sends a packet to itself with extension headers. This is based on gnrc_networking example.
+
+Enable debug output of `gnrc_ipv6.c` before run. When the test is run, it should show the following debug output:
```

Comment:
> Enable debug output of `gnrc_ipv6.c` before run.

might there be a way to not require that for the test? Because this does not allow for easy automation.